roles/nginx/templates/reverse-proxy

```diff
@@ -0,0 +1,14 @@
+server {
```

Comment:
can we make the location of this file match the location it will be moving to? ex: `templates/etc/nginx/sites-available/reverse-proxy`